src/rustdoc-internals/rustdoc-test-suite.md

```diff
@@ -78,34 +84,61 @@ determined by the XPath, and compares it to a pre-recorded value
 in a file. The file's name is the test's name with the `.rs` extension
 replaced with `.NAME.html`, where NAME is the snapshot's name.
 
-htmldocck supports the `--bless` option to accept the current subtree
+Htmldocck supports the `--bless` option to accept the current subtree
 as expected, saving it to the file determined by the snapshot's name.
 compiletest's `--bless` flag is forwarded to htmldocck.
 
+<!-- FIXME(fmease): Also mention that we normalize certain URLS
+both when and checking and when normalizing
+-->
+
 ### `has-dir`
 
 Usage: `//@ has-dir PATH`
 
-Checks for the existance of directory `PATH`.
+Checks for the existence of directory `PATH`.
 
 ### `files`
 
 Usage: `//@ files PATH ENTRIES`
 
 Checks that the directory `PATH` contains exactly `ENTRIES`.
-`ENTRIES` is a python list of strings inside a quoted string,
+`ENTRIES` is a Python list of strings inside a quoted string,
 as if it were to be parsed by `eval`.
 (note that the list is actually parsed by `shlex.split`,
-so it cannot contain arbitrary python expressions).
+so it cannot contain arbitrary Python expressions).
 
 Example: `//@ files "foo/bar" '["index.html", "sidebar-items.js"]'`
 
 [^1]: Whitespace normalization means that all spans of consecutive whitespace are replaced with a single space.  The files themselves are also whitespace-normalized.
 
+## Compiletest Directives
+
+In addition to the directives listed here,
+`rustdoc` tests also support most
+[compiletest directives](../tests/directives.html).
+
+<!-- FIXME(fmease):
+Should definitely also mention `//@ aux-crate` and `//@ proc-macro`
+UNLESS we nuke this paragraph entirely and refer to the compiletest section(s)?
+-->
```

Comment:
Remark (not-a-review-comment): I think this is where I would like compiletest to support some kind of directive handling "plugin" and allow htmldocck to register plugins. Then we could phrase this as "htmldocck registers additional directive handling to compiletest" either here, or even put this bit under main compiletest chapter.